configure.ac relies on git, which doesn't work for release tarball.
See: https://github.com/chaos/pdsh/issues/116
--- a/configure.ac	2017-06-28 19:28:26.000000000 -0500
+++ b/configure.ac	2019-09-11 08:41:27.613006239 -0500
@@ -4,12 +4,11 @@
 # Copyright (C) 2000-2002 Regents of the University of California
 # See ./DISCLAIMER
 #
 # This file is to be processed with autoconf to generate a configure script.
 
-AC_INIT([pdsh],
-        m4_esyscmd([git describe --always |  awk '/.*/ {sub(/^pdsh-/, ""); printf "%s",$1; exit}']))
+AC_INIT([pdsh], 2)
 
 AC_CONFIG_SRCDIR([pdsh])
 AC_CONFIG_AUX_DIR([config])
 AC_CONFIG_MACRO_DIR([config])
 AC_CONFIG_SRCDIR([src/pdsh/dsh.h])
